x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StopDataFrameAnalyticsAction.java

```diff
@@ -132,13 +132,11 @@ public void writeTo(StreamOutput out) throws IOException {
 
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-            builder.startObject();
-            if (id != null) {
-                builder.field(DataFrameAnalyticsConfig.ID.getPreferredName(), id);
-            }
-            builder.field(ALLOW_NO_MATCH.getPreferredName(), allowNoMatch);
-            builder.endObject();
-            return builder;
+            return builder
```

Comment:
Just a note that I don't think should change this but it's useful for the future.

I think the reason `toXContent` is implemented around the codebase mostly without the chained paradigm but with separating each step into its own statement is to make it easy to add statements that need to be surrounded with null checks. While the chained paradigm is nice and compact, if we need to add in the future a null-check for a certain field, we'll need to refactor the whole method.

Reply:
Ack